### ISSUE TITLE
Warning message with Android 14

### ISSUE THREAD

**Issue by toolstack** [schnatterer/nusic#30]
Current release on F-Droid displays warning message during first load:

![Screenshot_20240226-131146_nusic](https://github.com/schnatterer/nusic/assets/9053749/2720613c-6b12-4766-bcfe-a2c90434f396)

Presumably this is because it is targeting a too old SDK version.

**Comment by schnatterer** [comment]
@toolstack Thanks for letting me know.
To rebuild with latest SDK,most of the Android-related code would have to be rewritten, because of numerous depreciations. I won't be having time to do that any time soon.
 Fortunately, the general functionally of nusic is still there. It's running on my android 14 device.

**Comment by toolstack** [comment]
@schnatterer Yep, no issues so far that I've found either on 14, just the initial warning.

Presumably moving even one or two SDK versions forward would remove it.

**Comment by schnatterer** [comment]
@toolstack I think I actually tried that but it was no quick win 😐

**Comment by toolstack** [comment]
@schnatterer That's unfortunate.

When I get a chance I'll poke at it a bit as well.

**Comment by schnatterer** [comment]
Seems like with Android 15 nusic will not longer be installable:
> apps that target API levels lower than 24 (7.0) will not install with Android 15 in 2024.

https://bayton.org/android/android-14-minimum-sdk/
